src/utils.ts

```diff
@@ -56,6 +56,7 @@ export async function postSlackNotification (
   try {
     const statusIcon = status === 'success' ? '✅' : '❌'
     const afterSha = sha.slice(0, 7)
+    const dataDogDeploymentConfidenceDashboard = environment === 'production' ? 'https://app.datadoghq.com/dashboard/vbe-7ch-b6q/npm-inc-registry' : 'https://app.datadoghq.com/dashboard/v49-ema-xip/npm-inc-registry-staging'
```

Comment:
this action is public for now (when we implemented that, there was no possibility to run an action from the private repo), so i'd avoid hardcoding specific routes here if possible.

Another approach could be proceed with this change, but make the repo private, just need to check if we can use private repo actions now. I know that was on Actions roadmap.

Reply:
we could make this an env var and add it as an org secret

Reply:
Sounds good.